core/node/groups.go

```diff
@@ -313,7 +315,9 @@ func Offline(cfg *config.Config) fx.Option {
 		fx.Provide(offline.Exchange),
 		fx.Provide(DNSResolver),
 		fx.Provide(Namesys(0)),
-		fx.Provide(offroute.NewOfflineRouter),
+		fx.Provide(libp2p.Routing),
+		fx.Provide(libp2p.ContentRouting),
+		fx.Provide(libp2p.OfflineRouting),
```

Comment:
- `OfflineRouting` will create a special `routing.Routing`, the offline Router.
- `Routing` get all `routing.Routing` and creates a `TieredRouter` that will be used everywhere. On `Offline` mode it will be only the offline one.
- `ContentRouting` creates also a `TieredRouter` that will be used only for topic discovery. Topic discovery will be used for `PubSub` and it will be added as a `routing.Routing`.

I did that to create everything the same way on the `Offline` or `Online` mode. `ContentRouting` might be removed from here because we are not creating a `PubSub` router.